### PR TITLE
Add ansi-color support for windows

### PR DESCRIPTION
**Pull request by glkz**
Integrated [shiena/ansicolor](https://github.com/shiena/ansicolor).

We need to change color scheme a little bit. I am not a windows guy but as far as I can see windows shell does not support ansi codes like 90, 91. But codes like 30, 31, ... seem ok.

Taking #25 also into consideration, it may be good to rethink the built-in color scheme for larger platform support.

### CODE REVIEW

**Comment by glkz** [comment]
ping @astaxie. What do you think about merging this?

It doesn't break anything in windows right now. Just the default color-scheme should be re-arranged. I think it is a different story.

File: bat.go [hunk 1]
```diff
@@ -231,70 +232,39 @@ func main() {
...
-		if printV == "A" || printV == "H" || printV == "B" {
-			dump := httpreq.DumpRequest()
-			if printV == "B" {
-				dps := strings.Split(string(dump), "\n")
-				for i, line := range dps {
-					if len(strings.Trim(line, "\r\n ")) == 0 {
-						dump = []byte(strings.Join(dps[i:], "\n"))
-						break
-					}
+	fi, _ := os.Stdout.Stat()
```
Comment (astaxie) [inline bat.go]: in window it will cause error. Stdout is nil.

Comment (glkz) [inline bat.go]: I can compile and run this under windows 8 with go1.4.2.

Screenshot:
without-color

After setting appropriate colors(ansi color codes between 30-39):
with-color

**Comment by postromantic** [comment]
What was the reason not to merge this PR? I'm still using this fork https://github.com/glkz/bat/commits/feature-ansicolor_win on all my Windows machines (using bash for windows and cygwin) and it works well.

**Pull request closed without merging**